Mathlib/GroupTheory/Subgroup/Centralizer.lean

```diff
@@ -77,4 +77,22 @@ variable (H)
 theorem le_centralizer [h : H.IsCommutative] : H ≤ centralizer H :=
   le_centralizer_iff_isCommutative.mpr h
 
+/-- The homomorphism `Normalizer(H) → Aut(H)` with kernel `Centralizer(H)`. -/
+@[simps]
+def normalizer_monoidHom : H.normalizer →* MulAut H where
```

Comment:
Is this additivizable?

Reply:
No, because `MulAut` doesn't play well with `to_additive` (for instance, `MulAut` is a multiplicative group, and `AddAut` is also a multiplicative group).